### PR TITLE
Mayk/setup connections

### PR DESCRIPTION
**Pull request by MayHyeyeonKim**
## Description

1.1.1.5 Setup backend services and connections to airtable and any other services that the map would need to interface with
1.1.1.6 Build the backend model to interface with the map UI

I have created types for `generalInfo`, `hospitalInfo`, `hospitalRequest`, and `HospitalFunded`, and made interfaces for each of them.

I have also updated the env file as needed. Specifically, I replaced `VITE_AIRTABLE_BASE_ID_DAS=app6duHw2djMIOYnh` with `VITE_AIRTABLE_BASE_ID_GFL=appz9IKHnPPimu54m`. Please check it.

In `App.tsx`, I only created the interfaces for now, so in the useEffect's then block, I used `console.log` to verify if the data is coming in correctly. Your feedback would be appreciated.

I have commented out the code that I am not sure if it is needed.

Could you please review these changes? Thank you.

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

_Link user story from projects.digitalaidseattle.org_

- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well as any relevant images for UI changes._

### CODE REVIEW

File: coverage/.tmp/coverage-23.json
Comment (jnakaso) [inline coverage/.tmp/coverage-23.json:1]: please add coverage to .gitignore

File: .env
Comment (jnakaso) [inline .env:1]: There needs to be an empty line at the end of the file.

**Comment by jnakaso** [comment]
Please how the files can be separated into different sections (i.e. api, components, services, etc.).  Use Slack to start a discussion with other devs.

**Comment by jnakaso** [comment]
The build is failing.  Use `npm run build` locally before pushing.

**Review by jnakaso — CHANGES_REQUESTED** [review]
please address individual comments

File: .env
Comment (MayHyeyeonKim) [reply .env:1]: I added `'no-console': 'error',` in `.eslintrc.cjs` and set `"files.insertFinalNewline": true` in my local environment.

**Review by jnakaso — APPROVED** [review]
This looks good.

Let's consider adding a task for reorganizing files and removing example ones.

**Comment by jnakaso** [comment]
The data we are getting from GFL has no data in the fields attribute.  Let's followup up with Frank if that is right.

**Comment by Annysah** [comment]
> The data we are getting from GFL has no data in the fields attribute. Let's followup up with Frank if that is right.

Yes, I got undefined.

**Pull request merged**